### DISCUSSION TITLE
How can I integrate the Styling in my publish theme?

### DISCUSSION THREAD

**Discussion by aksel-a** [mayurankv/Obsidian-Code-Styler#269] [Q&A]
Hi there! This plugin is amazing! I'm working in my Obsidian page and would like to include the styling of the code blocks into my Publish theme, but I'm just a beginner in CSS, so I'm not sure about the best way to do it. Any initial recommendations?

**Comment by mayurankv** [comment]
Hiya, I know it's quite a while later so I'm sorry for the delay! I don't actually have Obsidian Publish so I'm unsure as to what work is required to make it compatible. Does the plugin currently not work at all in publish? Or does it partially show styling? How do other plugins fare? Is there one I should use for inspiration?

**Comment by mayurankv** [comment]
Refer to #309 hereon
